Examples/Temperature_Salinity_Diagram.ipynb

```diff
@@ -30,9 +30,11 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # todo delete:
</pre><p>Exactly what's todo here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
not sure, if nothing is clear then delete the comment!